### ISSUE TITLE
Ugly exceptions during invalid code parsing

### ISSUE THREAD

**Issue by Scony** [lark-parser/lark#1257]
**Describe the bug**

Ugly exceptions during invalid code parsing

**To Reproduce**

`pip install gdtoolkit` and then steps from: https://github.com/Scony/godot-gdscript-toolkit/issues/202

GDToolkit uses lark 1.1.5.

**Comment by MegaIng** [comment]
Can you please provide a minimal example? Working through an entire unfamiliar library and grammar is a lot of effort. If you provide a minimal self contained example, the process to fixing this will be massively speedup.

**Comment by Scony** [comment]
The thing is - I cannot reproduce it with fairly straightforward grammar.

**Comment by MegaIng** [comment]
Does it sill happen if you use the most recent master instead of the latest release?

**Comment by MegaIng** [comment]
It's probably a duplicate of the recently fixed #1154

**Comment by Scony** [comment]
@MegaIng you were right, with the following changes in `setup.py`:
```
-        "lark[regex]==1.1.5",
+        # "lark[regex]==1.1.5",
+        'lark[regex]@git+ssh://git@github.com/lark-parser/lark.git@2564232e0a6f2dbd15a25917129d3b773bd5e18f#egg=lark[regex]',
```
I could see the issue is no more. Therefore it's indeed a dup of #1154
